pandas/tests/frame/test_reshape.py

```diff
@@ -765,7 +765,8 @@ def test_unstack_unused_level(self, cols):
         tm.assert_frame_equal(result, expected)
 
     def test_unstack_nan_index(self):  # GH7466
-        cast = lambda val: "{0:1}".format("" if val != val else val)
+        val_str = lambda val: "" if val != val else val
```

Comment:
Hmm this is a little confusing. Do you even need the lambda? This is all local to the unstack function, no?

Reply:
What is your suggestion?

Reply:
Ah sorry I misunderstood before. Just `def cast` then instead of using a lambda - I think should make things easier

Reply:
Gotcha! Just pushed... Now waiting for the checks to run

Reply:
Can you move this into the body of the function? Shouldn't need a lambda here at all

Reply:
Sorry... I misunderstood the point.... Now I think I got it